README.md

```diff
@@ -28,13 +28,16 @@ What does it do?
 | CPython 3.7     | ✅ |   | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
 | CPython 3.8     | ✅ |   | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
 | CPython 3.9     | ✅ | ✅ | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
+| CPython 3.10¹     | ✅ | ✅ | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
 | PyPy 3.7 v7.3   | ✅ |   | ✅  |    | ✅ |    |    |    |   |
+<sup>¹ Requires `--pre` flag to be passed - Python Beta candidates not recommended for upload to PyPI. </sup><br>
```

Comment:
Should we expend a bit more on ABI compatibility here ? It might be too verbose for the README, maybe a link to an issue that's kept open until RC1 or Final Release with a bit more information ?